soc/xtensa/intel_adsp/common/soc_mp.c

```diff
@@ -354,12 +362,11 @@ void arch_start_cpu(int cpu_num, k_thread_stack_t *stack, int sz,
 void arch_sched_ipi(void)
 {
 #ifdef CONFIG_SOC_SERIES_INTEL_CAVS_V25
-	uint32_t prid;
+	uint32_t curr = prid();
```

Comment:
this seems to break bisect: `prid()` is only defined in a later commit https://github.com/zephyrproject-rtos/zephyr/pull/37647/commits/1eb62d4fc1ffd4ff2ae1528243d7ae6f2d9cd0c1